src/Uno.UI/UI/Xaml/Controls/Border/BorderLayerRenderer.Android.cs

```diff
@@ -31,6 +28,8 @@ partial class BorderLayerRenderer
 		private readonly SerialDisposable _layerDisposable = new SerialDisposable();
 		private static readonly float[] _outerRadiiStore = new float[8];
 		private static readonly float[] _innerRadiiStore = new float[8];
+		private static Paint _strokePaint;
+		private static Paint _fillPaint;
```

Comment:
Ok I see now why it's safe.

Reply:
It is the same as with `_outerRadiiStore` and `_innerRadiiStore` above. Because it has to run on the UI thread, there is currently no way for two `UpdateLayer` methods to be executing simultaneously, so these static fields should be safe. An issue would arise if we have multiple UI threads, but that is a non-supported scenario for now at least.